src/test/scala/com/campudus/tableaux/api/content/displayValues.scala

```diff
@@ -0,0 +1,154 @@
+package com.campudus.tableaux.api.content
+
+import com.campudus.tableaux.testtools.RequestCreation._
+import com.campudus.tableaux.testtools.TableauxTestBase
+import io.vertx.ext.unit.TestContext
+import io.vertx.ext.unit.junit.VertxUnitRunner
+import org.junit.Assert._
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.vertx.scala.core.json.{Json, JsonObject, JsonArray}
+
+@RunWith(classOf[VertxUnitRunner])
+class DisplayValueTest extends LinkTestBase {
+
+  val createTableJson = Json.obj("name" -> "Test Nr. 1")
+
+  val createNumberColumnJson =
+    Json.obj(
+      "columns" -> Json.arr(Json.obj("kind" -> "numeric", "name" -> "Test Column Number", "multilanguage" -> true)))
+
+  val createCurrencyColumnJson =
+    Json.obj(
+      "columns" -> Json.arr(Json.obj("kind" -> "currency", "name" -> "Test Column Currency", "multilanguage" -> true)))
+
+  val createBooleanColumnJson =
+    Json.obj(
+      "columns" -> Json.arr(Json.obj("kind" -> "boolean", "name" -> "Test Column Bool", "multilanguage" -> true)))
+
+  val createDateColumnJson =
+    Json.obj("columns" -> Json.arr(Json.obj("kind" -> "date", "name" -> "Test Column Date", "multilanguage" -> true)))
+
+  val createDateTimeColumnJson =
+    Json.obj(
+      "columns" -> Json.arr(Json.obj("kind" -> "datetime", "name" -> "Test Column DateTime", "multilanguage" -> true)))
+
+  val createTextColumnJson =
+    Json.obj("columns" -> Json.arr(Json.obj("kind" -> "text", "name" -> "Test Column Text", "multilanguage" -> true)))
+
+  val createValuePost = (value: Any) => {
+    Json.obj("value" -> value)
+  }
+  val putNumberValue = createValuePost(Json.obj("de-DE" -> 42.42))
+  val putCurrencyValue = createValuePost(Json.obj("DE" -> 1000.42))
+  val putBooleanValue = createValuePost(Json.obj("de-DE" -> true))
+  val putDateValue = createValuePost(Json.obj("de-DE" -> "2015-01-01"))
+  val putDateTimeValue = createValuePost(Json.obj("de-DE" -> "2015-01-01T13:37:47.110Z"))
+  val putTextValue = createValuePost(Json.obj("de-DE" -> "this is text"))
+
+  val expectedNumber = Json.obj(
+    "de-DE" -> "42.42",
```

Comment:
**[question]** - Wie wollen wir generell mit i18n formatting umgehen? Eigentlich sollte der value doch dann hier auch in `42,42` formatiert werden. Bei currency wird das scheinbar schon gemacht. Bei boolean nicht.

@Zwergal @hermann-p 
Bleibt die generelle Frage, wie weit man da gehen will. Ich glaube fast das müssen wir diskutieren. 
Bei atomic values könnte man den Rest auch vom Frontend erledigen lassen, sehe ich aber nicht als sinnvoll an, wenn dann ganz vom Backend. 
Wenn der value aber wie im Falle von Link Spalten concateniert wird, geht das hinterher im Frontend eh nicht mehr. Imho müsste deshalb eigentlich aller Frontend-Formatierungs-Schmuh für die dispayValues auch ins backend gebracht werden. Evtl. muss man sogar noch einen zusätzlichen local value dafür mit ins Backend geben. Da bin ich mir nicht sicher, das wisst ihr vermutlich besser.